### PR TITLE
[4.0] pacemaker: added support for op defaults

### PR DESCRIPTION
**Pull request by AbelNavarro**
backport of https://github.com/crowbar/crowbar-ha/pull/297

We needed to add a default option for the pacemaker op defaults. In
particular we are interested in offering a default for "op monitor
on-fail" action.

Pacemaker does not offer a configuration option to offer such defaults
for all resources. There's a way to do it for a single resource with
rsc_defaults, but that is not our case. Also with Pacemaker you could
link one resource operations to another one with 'id-ref', but in that
case if you modify the reference resource it will cascade changes to all
other resourced.

This approach allows to store in the pacemaker barclamp a default for
op. In this case we are only interested in op monitor but could easily
be extended later on.

The primitive will read that value from the founder node and set it to
all 'op monitor on-fail' actions. Should the user had configured
manually the defaults (editing the barclamp), then those manually
configured values will prevail.

(cherry picked from commit e3a6e4c1445986e9688c50fc065afea5a56de71e)

### CODE REVIEW

**Comment by AbelNavarro** [comment]
[Rework on-fail for stop actions setting in pacemaker](https://trello.com/c/Jkr9NDFX/112-rework-on-fail-for-stop-actions-setting-in-pacemaker)

**Comment by aspiers** [comment]
@AbelNavarro Please can you follow [the established process for handling backports](https://etherpad.nue.suse.com/p/cloud-PR-porting)?  (Yes, I know we need to turn this into a proper document in GitHub. Volunteers welcome.)

**Comment by AbelNavarro** [comment]
@aspiers thanks for the feedback. I made the changes as suggested by the backport guidelines.

Notice that there are no broken URLs in the description :-P

File: chef/data_bags/crowbar/template-pacemaker.json [hunk 1]
```diff
@@ -54,14 +54,19 @@
         "admin_name": "",
         "public_name": ""
       },
-      "clone_stateless_services": true
+      "clone_stateless_services": false,
```
Comment (vuntz) [inline chef/data_bags/crowbar/template-pacemaker.json]: Careful: you're changing the default here! It's different in stable/4.0.

Comment (AbelNavarro) [reply chef/data_bags/crowbar/template-pacemaker.json]: nice catch, fixing that.

**Comment by AbelNavarro** [comment]
@aspiers mind having another look at this PR?

**Pull request merged**